dateofdelivery.php

```diff
@@ -43,7 +43,7 @@ public function __construct()
 		parent::__construct();
 
 		$this->displayName = $this->trans('Date of delivery', array(), 'Modules.Dateofdelivery.Admin');
-		$this->description = $this->trans('Displays an approximate date of delivery', array(), 'Modules.Dateofdelivery.Admin');
+		$this->description = $this->trans('72% of customers are satisfied when they can see expected delivery dates of their purchases, let’s satisfy yours up to 100%!', array(), 'Modules.Dateofdelivery.Admin');
```

Comment:
Shouldn't be `let's`  instead of `let’s `
```suggestion
		$this->description = $this->trans('72% of customers are satisfied when they can see expected delivery dates of their purchases, let's satisfy yours up to 100%!', array(), 'Modules.Dateofdelivery.Admin');
```